services/src/api/model/responses/datasets/errors.rs

```diff
@@ -50,3 +50,21 @@ impl ResponseError for GetDatasetError {
         StatusCode::BAD_REQUEST
     }
 }
+
+#[derive(Debug, Snafu, IntoStaticStr)]
+#[snafu(visibility(pub(crate)))]
+#[snafu(context(suffix(false)))] // disables default `Snafu` suffix
+pub enum UpdateDatasetError {
+    CannotLoadDatasetForUpdate { source: error::Error },
+    CannotUpdateDataset { source: error::Error },
```

Comment:
Please add a display for the errors.

Reply:
[d10f183](https://github.com/geo-engine/geoengine/pull/934/commits/d10f18383d01222dae649c56b3a16652b69f2b65)

Don't you want to add the reason (source)?

In theory, it should only display things for the user.

Reply:
the source is currently a tokio postgres error. I would have to do build more error handling into it to give the source.